astropy/table/tests/test_table.py

```diff
@@ -2105,3 +2105,11 @@ def test_qtable_read_for_ipac_table_with_char_columns():
     t1.write(out, format="ascii.ipac")
     t2 = table.QTable.read(out.getvalue(), format="ascii.ipac", guess=False)
     assert t2["B"].unit is None
+
+
+def test_create_table_from_final_row():
+    """Regression test for issue #8422: passing the last row of a table into
+    Table should return a new table containing that row."""
+    t = table.Table([(1, 2)], names=['col'])
+    row = t[-1]
+    assert(table.Table(row)['col'][0] == 2)
```

Comment:
moving the new table creation out to a line would make this a little bit cleaner. also remove the parens.